### ISSUE TITLE
loophole in cash aquisition

### ISSUE THREAD

**Issue by bczretribution** [AaronGullickson/mekhq#45]
### Environment
0.49.49-SNAPSHOT
Java 11
Win 11

### Description
Claiming salvage allows to "auto ransom", leading to gaining C-Bill funds.  Storyarc text implies that this is not desired and player is only able to work with what they have to start with and what they can scavenge.  It is left unclear though as to if player should have access to cash, and should be able to purchase desired parts from part market.  

Additionally, unclear if player should be allowed to hire new staff, but personnel market is VERY well stocked on day one.
![image](https://github.com/AaronGullickson/mekhq/assets/34722287/e0fd165d-92bc-4997-bbcd-5fcd49fe4464)

**Comment by AaronGullickson** [comment]
I am not familiar with the auto-ransom feature, but I will check to see if it can be turned off. Presently, the personnel market cannot be turned off, but I am planning to change that.

**Comment by AaronGullickson** [comment]
I put in a PR to disable personnel markets. Once that goes through I will change the initial campaign. I don't see anywhere the ability to ransom units. Technically, it shouldn't really affect anything because the purchasing of equipments and units is shut off. So you can get money, but what are you going to spend it on? 

I am also going to be adding a narrative element explaining what options can be changed and what should not be changed at the start of the story arc.

**Comment by bczretribution** [comment]
On the post battle screens, when looking at the list of enemy units, there are 3 check boxes, salvage, escaped, and ransom.  Selecting ransom automatically sells the unit for you, a QOL that was added a while back.

If everything else is locked down its still an issue.  if a player is unfamiliar with the mechanics of the story arc, they could ransom units (in my case 4 that were non-repairable), anywhere up to and including 100% of their salvage, which would then mean they have lost out on important resources in the way of spare parts/new mechs.  Adding info on the options setup for the arc would definitely help inform players so they are less likely to make the mistake, but if possible it would be better if it was locked down like the personnel market and parts market.

TL:DR It isn't about the money, its about the loss of spare parts and mechs.

**Comment by AaronGullickson** [comment]
Ahh, apparently my eye just skips right over that. Odd name as well. Seems like "Sold" would be a more intuitive label than "Ransom." 

I would need to add a campaign option to MHQ to disable it, so a bit more work but do-able.

**Comment by AaronGullickson** [comment]
Thought about this a little more and there is already a campaign option to enable or disable selling of units. If selling is disabled, then the "Ransom" button should also be disabled. This should be a relatively easy fix. I put up an issue for it at MegaMek/mekhq#3955

**Comment by AaronGullickson** [comment]
Campaign options changed in cb3b88526b.
